.werft/platform-trigger-artificial-job.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# The easiest way to run this script is through Werft so you don't have to worry
+# about installing the appropraite service account etc. locally.
+#
+#   werft job run github -j .werft/platform-trigger-artificial-job.yaml -s .werft/platform-trigger-artificial-job.sh
+#
+
+sleep 1
+
+set -Eeuo pipefail
+
+# Activate service account and install core-dev context
```

Comment:
Do we need this? The werft CLI should be using gRPC so we shouldn't need access to the core-dev cluster for that anymore at least ☺️

Reply:
aaaaah good point. It will just work out-of-the-box? No need to configure anything with grpc?